src/schema/tags.ts

```diff
@@ -89,6 +96,14 @@ export const typeDefs = /* GraphQL */ `
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export const resolvers: IResolvers<any, Context> = traceResolvers({
   Query: {
+    tags: (_, __, ctx, info): Promise<GQLKeyword[]> =>
+      graphorm.query<GQLKeyword>(ctx, info, (builder) => {
+        builder.queryBuilder = builder.queryBuilder.where({
+          status: 'allow',
+        });
```

Comment:
limit?

Reply:
Since it should render all in one visual I decided not to add limit to it 😅

Reply:
it will list 827 of them as currently stands, I think we should add upper limit just in case. Maybe 1000?

Reply:
Not sure what it would add to be honest?
We don't get many new ones, and even if we do get 1001 we want to render them all right?

Reply:
I did add it :)